doc/source/examples/cfrl_adult.ipynb

```diff
@@ -28,19 +28,33 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "!pip install xgboost"
+    "<div class=\"alert alert-info\">\n",
+    "To enable support for CounterfactualRLTabular with tensorflow backend, you may need to run\n",
+    "    \n",
+    "```bash\n",
+    "pip install alibi[tensorflow]\n",
+    "```\n",
+    "\n",
+    "</div>"
```

Comment:
Same comment as above, can we make the box title be "Note" and the text part of the body?